sqladmin/application.py

```diff
@@ -103,6 +117,27 @@ class UserAdmin(ModelAdmin, model=User):
 
         self._model_admins.append((model()))
 
+    def register_view(self, view: Type["ModelView"]):
+
+        class_view = view()
+        class_view.name_plural = class_view.name
+        class_view.url_path_for = self.app.url_path_for
+
+        # add template directory
+        class_view.templates = self.init_templating_engine(
+            extra_loader=FileSystemLoader(class_view.template_path)
```

Comment:
Maybe it's more useful to accept one `templates_path` with `Admin(...)` instead of setting it per view?